build.gradle

```diff
@@ -83,12 +83,12 @@ allprojects {
 }
 
```

Comment:
@eirsep please change source / compatibility level as well, since 2.0 has JDK-11 baseline 

```
sourceCompatibility = 11
targetCompatibility = 11
```

Reply:
@reta I have just merged #114 which had updated the compatibility to JDK-11